tests/models.py

```diff
@@ -56,7 +56,7 @@
 from airflow.utils.trigger_rule import TriggerRule
 from mock import patch, ANY
 from parameterized import parameterized
-from tempfile import NamedTemporaryFile
+from tempfile import NamedTemporaryFile, mkdtemp
```

Comment:
small nit: could you move mkdtemp before NamedTemporaryFile given we put lower case import before upper case(L57)?

Reply:
Updated as suggested. Thanks.